examples/speech_recognition/SpeechRecognitionExample.js

```diff
@@ -0,0 +1,224 @@
+class SpeechRecognitionExample extends BaseMircophoneExample {
+  constructor(models) {
+    super(models);
+    this.targetMax = 16384;
```

Comment:
@miaobin @mingmingtasd @fujunwei Is there some reference for this number **16384** ?

Reply:
In static quantization mode, the first utterance in the input ARK file is scanned for dynamic range. The scale factor (floating point scalar multiplier) required to scale the maximum input value of the first utterance to 16384 (15 bits) is used for all subsequent inputs.

Reply:
Please add comment here and if there is C++ example, plesae also refer to that. Thanks.